config/locales/en.yml

```diff
@@ -486,3 +486,9 @@ en:
     transparency: See all our transparency data
     written_statement: See all our written statements to Parliament
     authored_article: See all our authored articles
+  broken_links:
+    title: Broken links
+    broken: 
+      subheading: We’ve found links in this document that may be broken
+    caution:
+      subheading: We've found some issues with links that may indicate problems, you should apply caution in linking to them
```

Comment:
I think we should use fancy quotes here like in the broken subheading: `We’ve`

Reply:
Ah sure, in the broken subheading the `We’ve` has got a curly quote compared to the `We've` in the caution subheading. We should make it so the caution also has that curly quote. You should be able to just copy and paste it from the broken subheading.